src/html/logo.html

```diff
@@ -4,7 +4,7 @@ <h1 class="h1 weight-strong">Logo</h1>
   <h2 class="h2 weight-strong">Icon</h2>
   <div class="figure-wrapper">
     <figure class="grid-block figure vertical shrink figure-logo-icon">
-      <img class="img" src="images/logo/pink-icon/RGB/runnable-icon-pink-rgb.svg" width="190" height="190" alt="Lockup Purple">
+      <img class="img" src="images/logo/pink-icon/RGB/runnable-icon-pink-rgb.svg" width="292" alt="Lockup Purple">
```

Comment:
All images should have both `width` and `height` attributes. Let's go ahead and add any attributes that are missing to the logos/icons and bears.